### PR TITLE
Versioning documentation: Reordering documentation to make tspconfig.yaml change more visible 

### PR DESCRIPTION
**Pull request by AlitzelMendez**
Working on this issue: https://github.com/Azure/typespec-azure/issues/1397

The changes are done on `tspconfig.yaml`  but is not the first visible tab so is difficult to observe where the changes are done, so all 3 scenarios look the same, reordering this to show on the first tab the file in where the changes are done.
 
Creating a draft as I need to corroborate if more changes are needed.

### CODE REVIEW

File: docs/howtos/Client Generation/09versioning.mdx [hunk 1]
```diff
@@ -258,11 +258,29 @@ serviceClientClient.stableFunctionality(stableModel);
 
 ### Override to a specific version
 
-You can override the signature to return the api surface area for a specific api version.
+You can override the signature to return the api surface area for a specific api version, this change is done in `tspconfig.yaml`.
```
Comment (markcowl) [inline docs/howtos/Client Generation/09versioning.mdx:261]: I think I would add this after the sentence below about the example, and make it more parallel with the example below, with a similar sentence like 'In this example, you cna see how this change is made in tspconfig.yaml'

File: website/src/content/docs/docs/howtos/Client Generation/09versioning.mdx [hunk 2]
```diff
@@ -354,12 +354,28 @@ serviceClientClient.previewFunctionality(previewModel);
...
-You can also override the signature to return the combined api surface area of all of the separate api versions. Different languages have different support for versioning validation
+You can also override the signature to return the combined api surface area of all of the separate api versions. Different languages have different support for versioning validation.
+
+In the following examples, you can observe how this change is made in `tspconfig.yaml`.
 
 <ClientTabs>
 
 <ClientTabItem lang="typespec">
 
+```yaml
```
Comment (timotheeguerin) [inline website/src/content/docs/docs/howtos/Client Generation/09versioning.mdx:365]:
```suggestion
```yaml title=tspconfig.yaml
```

can you add that to all of them will make it clearer where teh file is

File: website/src/content/docs/docs/howtos/Client Generation/09versioning.mdx [hunk 3]
```diff
@@ -354,12 +354,28 @@ serviceClientClient.previewFunctionality(previewModel);
...
+options:
+"@azure-tools/typespec-python":
```
Comment (timotheeguerin) [inline website/src/content/docs/docs/howtos/Client Generation/09versioning.mdx:367]: I think there is some indentation that got dopped here

File: website/src/content/docs/docs/howtos/Client Generation/09versioning.mdx (hunk 2 above)
Comment (AlitzelMendez) [reply website/src/content/docs/docs/howtos/Client Generation/09versioning.mdx:365]: thanks Tim!!!

**Review by timotheeguerin — APPROVED** [review]

**Review by allenjzhang — APPROVED** [review]

**Pull request merged**